webroot/faq/faq.md

```diff
@@ -150,6 +150,11 @@ A: Instead of just one whitepaper, we're going to have academic papers in journa
 
 
 
++ <b>Q: Who is in the Chia team?</b>
+
+A: Bram Cohen: @bramcohen, Ryan Singer: @ryan_singer , Richard Kiss: @richardkiss and Alex Wice: @alexwice 
```

Comment:
Spoke to Ryan Singer about updating the homepage with Team, members, their titles and their Keybase handles. You may want to add Twitter handles, but I presume you want communication about Chia on Keybase. This decision is up to the team. Just thought it was important that interested parties could see who members were so that they could appreciate any announcements of Keybase chat as being from an employee / founder of Chia.

Reply:
That sounds right!. What I meant is that for general public though, it might seem these are Twitter handles, so a clarification in this regard would I think be good :)